fuel/app/classes/controller/test.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class Controller_Test extends Controller
+{
+  public function action_ci($arg = NULL)
```

Comment:
__[code_sniffer]__
* Tabs must be used to indent lines; spaces are not allowed
* TRUE, FALSE and NULL must be lowercase; expected "null" but found "NULL"